cmd/chunk/main.go

```diff
@@ -4,18 +4,54 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"time"
 
 	"github.com/cuducos/chunk"
+	"github.com/spf13/cobra"
 )
 
-func main() {
-	chunk := chunk.DefaultDownloader()
-	prog := newProgress()
-	for status := range chunk.Download(os.Args[1:len(os.Args)]...) {
-		if status.Error != nil {
-			log.Fatal(status.Error)
+var rootCmd = &cobra.Command{
+	Use:   "chunk",
+	Short: "Download tool for slow and unstable servers",
+	Long:  `The idea of the project emerged as it was difficult for Minha Receita to handle the download of 37 files that adds up to just approx. 5Gb. Most of the download solutions out there (e.g. got) seem to be prepared for downloading large files, not for downloading from slow and unstable servers — which is the case at hand.`,
```

Comment:
Not sure this context is useful here. What about:

> Download tool for slow and unstable servers using HTTP range requests, retries per HTTP request (not by file), prevents re-downloading the same content range and supports wait time to give servers time to recover.

NIT any reasons to use ` instead of "?